mne/event.py

```diff
@@ -885,3 +886,349 @@ def concatenate_events(events, first_samps, last_samps):
+        refEvents_t = times[refEvents_inds]
+        if cat_reqev:
+            # indices of times where req. event occurs
+            reqEvents_inds = np.where(events[:, 2] & (
+                1 << cat_reqev - 1))[0]
+            reqEvents_t = times[reqEvents_inds]
+            # relative (to refevent) time window where req. event
+            # must occur (e.g. [0 .2])
+            twin = [0, (-1)**(cat['reqwhen']) * cat['reqwithin']]
+            win = np.round(np.array(sorted(twin)) * sfreq)  # to samples
+            refEvents_wins = refEvents_t[:, None] + win
+            req_acc = np.zeros(refEvents_inds.shape, dtype=bool)
+            for t in reqEvents_t:
+                # mark time windows where req. condition is satisfied
+                reqEvent_in_win = np.logical_and(
+                    t >= refEvents_wins[:, 0], t <= refEvents_wins[:, 1])
+                req_acc |= reqEvent_in_win
+            # drop ref. events where req. event condition is not satisfied
+            refEvents_inds = refEvents_inds[np.where(req_acc)]
+            refEvents_t = times[refEvents_inds]
+        # adjust for trigger-stimulus delay by delaying the ref. event
+        refEvents_t += int(np.round(self._events[cat_ev]['delay'] * sfreq))
+        return refEvents_t
+
+    @property
+    def categories(self):
+        """ Return list of averaging categories in DACQ defined order. Only
+        returns categories marked active in DACQ. """
+        cats = sorted(self._categories_in_use.values(),
+                      key=lambda cat: cat['index'])
+        return cats
+
+    @property
+    def events(self):
+        """ Return events in DACQ defined order. Only returns events
+        that are in use (referred to by a category). """
+        evs = sorted(self._events_in_use.values(),
+                     key=lambda ev: ev['index'])
+        return evs
+
+    @property
+    def event_names(self):
+        return [event['comment'] for event in self.events]
+
+    def get_epochs(self, raw, category, picks=None, reject=None, flat=None,
+                   baseline=(None, 0), detrend=None, stim_channel=None,
+                   mask=None, uint_cast=None, mask_type=None):
+        """ Get an mne.Epochs instance corresponding to the given DACQ
+        (data acquisition) category.
+
+        Parameters
+        ----------
+        raw : Raw object
+            An instance of Raw.
+        category : string | dict
+            The category to use. Can be a name (e.g. 'my_category) or
+            category dict  (e.g. eav['my_category'], where eav is an instance
+            of ElektaAverager).
+        picks : array-like of int | None (default)
+            Indices of channels to include (if None, all channels are used).
+        reject : True | dict | None
+            Rejection parameters based on peak-to-peak amplitude.
+            Valid keys are 'grad' | 'mag' | 'eeg' | 'eog' | 'ecg'.
+            If reject is None then no rejection is done. Example::
+
```

Comment:
BTW the extra lines were also in `epochs.py`, I just copied the reject docstring from there.